storagedriver.go

```diff
@@ -31,18 +34,28 @@ import (
 	"github.com/golang/glog"
 )
 
-var storageDuration = flag.Duration("storage_duration", 2*time.Minute, "How long to keep data stored (Default: 2min).")
+var (
+	storageDriver   string
+	storageDuration = flag.Duration("storage_duration", 2*time.Minute, "How long to keep data stored (Default: 2min).")
+)
+
+func init() {
+	// Add storage driver flag.
+	options := storage.ListDrivers()
+	sort.Strings(options)
+	storageDriverUsage := fmt.Sprintf("Storage `driver` to use. Data is always cached shortly in memory, this controls where data is pushed besides the local cache. Empty means none. Options are: <empty>, %s", strings.Join(options, ", "))
+	flag.StringVar(&storageDriver, "storage_driver", "", storageDriverUsage)
```

Comment:
Because I wanted to dynamically create the usage message. Options for making it more consistent:
1. Keep storageDriver a `*string`, and use the `flag.String` method
2. Move storageDuration initiallization into init()
3. Make a function for getting the sorted options, inline the usage creation

WDYT?